src/components/Blue.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { SeatBook } from "../styles/Search.styled";
+
+const Blue = () => {
```

Comment:
Reduce the code repetition of having same markup multiple times by changing only the style of the seat component based on the state.